### DISCUSSION TITLE
npz training dataset has different cells in PBC

### DISCUSSION THREAD

**Discussion by afour9961207** [mir-group/nequip#384] [Q&A]
Hello, 

I'm considering training with the PBC dataset, but my cells may vary. 
According to previous [discussion](https://github.com/mir-group/nequip/discussions/242?fbclid=IwAR1VCao_m25ZgjPYDRFbfCXG2K0e_MsPlLdCSFtZ7FZ1_pClDS_Ceai1Xn8), if my training data doesn't have the same cell, I should place the PBC and cell tags in the `key_mapping` (not in `npz_fixed_field_keys`). 
They should have `pbc = (True, True, True)` in the total shape of (nframe, 3) and `cell = [[19.57, 0, 0], [0, 19.57, 0], [0, 0, 19.57]]` in the total shape of (nframe, 3, 3). 
![Screenshot 2023-11-13 132732](https://github.com/mir-group/nequip/assets/48210429/156b6644-0ed7-4992-9863-34acba29ccd9)

Am I misunderstanding anything, or is this a correct setup? 

Thanks for your clarification!

**Comment by Linux-cpp-lisp** [comment]
If I understand right this is fine, but you have to map to the actual key name (`pbc` or `cell`), not this plain english description of the field.
